roles/ironic-control/tasks/main.yml

```diff
@@ -1,91 +1,35 @@
 ---
-- name: install ironic pxe driver prereqs
-  apt: pkg={{ item }} state=installed
-  with_items:
-    - tftpd-hpa
-    - tftp
-    - syslinux
-    - syslinux-common
-  register: result
-  until: result|succeeded
-  retries: 5
-
-- name: create tftpboot
-  file: path={{ ironic.tftpboot_path }}
-        state=directory
-        owner=ironic
-        group=ironic
-        mode=0755
-
-- name: check if pxelinux.0 exists
-  stat: path={{ ironic.tftpboot_path }}/pxelinux.0
-  register: pxe
-
-- name: copy pxelinux.0 to tftpboot
-  command: cp /usr/lib/syslinux/pxelinux.0 {{ ironic.tftpboot_path }}/
-  when: pxe.stat.exists == False
-
-- name: create tfpd map file
-  copy: src=tftpboot/map-file dest=/tftpboot/map-file
-
-- name: create tftpd defaults
-  template: src=etc/default/tftpd-hpa dest=/etc/default/tftpd-hpa
-  notify: restart tftpd
-
 - name: install ironic controller services
   upstart_service: name={{ item }}
                    user=ironic
                    cmd=/usr/local/bin/{{ item }}
                    config_dirs=/etc/ironic
   with_items:
     - ironic-api
-    - ironic-conductor
 
 - name: stop ironic services before db sync
   service: name={{ item }} state=stopped
   with_items:
     - ironic-api
-    - ironic-conductor
   tags: db-migrate
 
+# FIXME(deva)
+# This dbsync is running only on certain hosts, when in actuality
+# it might want to coordinate with all the ironic services
 - name: sync ironic database
-  command: "/usr/local/bin/ironic-dbsync upgrade"
+  command: ironic-dbsync
   when: database_create.changed or force_sync|default('false')|bool
   changed_when: true
   run_once: true
-  notify: restart ironic controller services
+  notify: restart ironic services
   # we want this to always be changed so that it can notify the service restart
   tags: db-migrate
 
-- name: trigger restart on upgrades
-  debug:
-    msg: "Triggering service restart for upgrade"
-  changed_when: True
-  notify: restart ironic services
-  when: code_has_changed | default('False') | bool and
-        upgrade | default('False') | bool
-
-- meta: flush_handlers
```

Comment:
Why was this taken out? We need to be able to trigger the service to restart in various scenarios.